tensorflow/contrib/crf/python/kernel_tests/crf_test.py

```diff
@@ -114,7 +114,7 @@ def testCrfLogNorm(self):
         np.array([[4, 5, -3], [3, -1, 3], [-1, 2, 1], [0, 0, 0]],
                  dtype=np.float32),
         np.array([[3, -1, 3]],
-                 dtype=np.float32),
+                 dtype=np.float32)
```

Comment:
This change doesn't seem necessary?
In general, I'd suggest that we avoid these one-character formatting changes unless there are other more significant changes to the file (which isn't the case here).